DEVELOPMENT.md

```diff
@@ -0,0 +1,101 @@
+
+### Server
+
+To run the server in GoLand:
+1. Create a new "Go Build" configuration.
+2. In the "Program Arguments" section, enter `server --dev-mode`.
+
+Then you're ready to run or debug this target.
+
+### Admin UI
+
+To run the Admin UI in GoLand or Web Storm:
+1. Create a new "npm" build.
+2. In the "package.json" dropdown, select the `package.json` file in the `ui/admin` directory.
+3. In the "Command" dropdown, select `run`.
+4. In the "Scripts" dropdown, select `dev`.
+5. In the "Environment" section, enter `VITE_API_IN_BROWSER=true`.
+
+Then you're ready to run or debug this target.
+
+### User UI
+
+To run the User UI in GoLand or Web Storm:
+1. Create a new "npm" build.
+2. In the "package.json" dropdown, select the `package.json` file in the `ui/user` directory.
+3. In the "Command" dropdown, select `run`.
+4. In the "Scripts" dropdown, select `dev`.
+5. In the "Environment" section, enter `VITE_API_IN_BROWSER=true`.
+
+Then you're ready to run or debug this target.
+
+## Developing Obot Tools
+
+Obot has a set of packaged tools. These tools are in the repo `github.com/obot-platform/tools`. By default, Obot will pull the tools from this repo. However, when developing tools in this repo, you can follow these steps to use a local copy.
+
+1. Clone `github.com/obot-platform/tools` to your local machine.
+2. In the root directory of the tools repo on your local machine, run `make build`.
+3. Run the Obot server, either with `make dev` or in your IDE, with the `OBOT_SERVER_TOOL_REGISTRY` environment variable set to the root directory of the tools repo.
+
+Now, any time one of these tools is run, your local copy will be used.
+
+> [!IMPORTANT]
+> Any time you change a Go based tool in your local repo, you must run `make build` in the tools repo for the changes to take effect with Obot.
+
+> [!NOTE]
+> Tool definitions and metadata are only synced to Obot every hour. Therefore, if you make a change to the tool in your local machine, it may not reflect immediately in Obot. Rest assured that the latest version is used when running the tool.
+
+## Obot Server Dev Mode
+
+In the description above for running the server in an IDE, the `--dev-mode` flag is used. This flag is also used when running the server with `make dev`. This does a few things (like turns on debug logging), the most helpful of which is to give you access to the database via `kubectl`. The kubeconfig is located at `tools/devmode-kubeconfig`.
+
+For example, from the root directory of the obot repo, you can list all agents in your setup with `kubectl --kubeconfig tools/devmode-kubeconfig get agents`.
+
+## Obot Credentials
+
+The GPTScript credentials for Obot are, by default, store in an SQLite database called `obot-credentials.db` in the root of the obot repo. You can use the `sqlite3` CLI to inspect the database directly: `sqlite3 obot-credentials.db`.
```

Comment:
```suggestion
The GPTScript credentials for Obot are, by default, stored in a SQLite database called `obot-credentials.db` in the root of the obot repo. You can use the `sqlite3` CLI to inspect the database directly: `sqlite3 obot-credentials.db`.
```

I know it's probably technically pronounced "S Q lite", but I'm pretty sure most people just say "sequelite".

Reply:
Done.